node/node.go

```diff
@@ -443,6 +449,15 @@ func (n *Node) getTokensIDsWithFilter(
 		return nil, api.BlockInfo{}, ErrMetachainOnlyEndpoint
 	}
 
+	return n.baseGetTokensIDsWithFilter(f, options, ctx)
+}
+
+// baseGetTokensIDsWithFilter returns token IDs based on filters
```

Comment:
Can delete this, we only add comments for exported functions. If the func name doesn't add any value to the func description, it can be removed